.NET/Microsoft.Recognizers.Text.NumberWithUnit/French/Extractors/FrenchNumberWithUnitExtractorConfiguration.cs

```diff
@@ -50,6 +50,8 @@ protected FrenchNumberWithUnitExtractorConfiguration(CultureInfo ci)
 
         public virtual Regex AmbiguousUnitNumberMultiplierRegex => null;
 
+        public virtual Regex HalfUnitRegex => null;
```

Comment:
The move to a method in a Chinese config would eliminate all these unused regex references too.

Reply:
Same as previous discussion, this part is not necessary for now and has been removed.